tests/test_configurations/test_base.py

```diff
@@ -1005,7 +1005,7 @@ def test_package_version_lt():
     """Test package version __lt__."""
     v1 = PackageVersion("0.1.0")
     v2 = PackageVersion("0.2.0")
-    v3 = PackageVersion("latest")
+    v3 = PackageVersion("0.12.0")
     assert v1 < v2 < v3
```

Comment:
NOTE: we do not delegate to `semver.VersionInfo` to remain the ability to use `"latest"` and `"any"`, however we cannot compare `PackageVersion("1.0.0") < PackageVersion("latest")`, which will raise `ValueError: latest is not valid SemVer string` due to the deflection. 

see [`Version.__lt__`](https://github.com/python-semver/python-semver/blob/3ec0131ea4301d92de95a12601abdcc86675d465/src/semver/version.py#L438-L440) and [`Version.compare`](
https://github.com/python-semver/python-semver/blob/3ec0131ea4301d92de95a12601abdcc86675d465/src/semver/version.py#L353-L354)

Reply:
Having this change does not make sense at all, the `latest` version should always be the superior when comparing to other versions.

Reply:
Au contraire mon ami.

- This change would have made the original implementation fail.
- Comparing "latest" should not always return `True`, in case latest is equal to current version being compared to. In this sense "latest" is a property.
- We also have this "any" which poses another issue. This, however, is simpler since the comparison operator can easily be deduced to have to return `False`.

Hence, additional tests are required to cover these cases, for which I can stack a PR later this week. Prior to that, however, I need to know how we determine what the current latest version is. It should be smooth sailing from there.